CMakeLists.txt

```diff
@@ -435,28 +436,38 @@ if(USE_MPI)
 endif(USE_MPI)
 
 ###############################################################################
-# Find MKL
+# Find BLAS library
 if(COMPILE_CPU)
-  if(USE_MKL)
-    find_package(MKL)
-  endif(USE_MKL)
-  if(MKL_FOUND)
-    include_directories(${MKL_INCLUDE_DIR})
-    set(EXT_LIBS ${EXT_LIBS} ${MKL_LIBRARIES})
-    set(BLAS_FOUND TRUE)
-    add_definitions(-DBLAS_FOUND=1 -DMKL_FOUND=1)
-  else(MKL_FOUND)
-    set(BLAS_VENDOR "OpenBLAS")
-    find_package(BLAS)
-    if(BLAS_FOUND)
-      include(FindCBLAS)
-      if(CBLAS_FOUND)
-        include_directories(${BLAS_INCLUDE_DIR} ${CBLAS_INCLUDE_DIR})
-        set(EXT_LIBS ${EXT_LIBS} ${BLAS_LIBRARIES} ${CBLAS_LIBRARIES})
-        add_definitions(-DBLAS_FOUND=1)
-      endif(CBLAS_FOUND)
-    endif(BLAS_FOUND)
-  endif(MKL_FOUND)
+  if(USE_APPLE_ACCELERATE)
+    if(NOT APPLE)
+      message(FATAL_ERROR "FATAL ERROR: Apple Accelerate only works on macOS.")
+    endif()
+    set(BLAS_VENDOR "Accelerate")
+    include_directories("/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/Accelerate.framework/Frameworks/vecLib.framework/Headers")
```

Comment:
Is this the default path where Accelerate is installed? Could you maybe add in a comment how to install it and/or a link to the framework, which hopefully can help others to quickly start with this?

Reply:
I can add the above link in a comment. it's bundled with macOS, so there's nothing to install, and the idea is that Accelerate just implements the BLAS interface, so it's a transparent change that doesn't require users to do/learn anything.

btw I'm still blocked from merging this because of the earlier change request.

Reply:
Great, thanks! No problem, we will merge it.